TestPlan.md

```diff
@@ -144,9 +154,15 @@ The tree view of maven projects should update when any pom.xml is created/modifi
 3. Verify:
     1. The corresponding project item is removed from the sidebar.
 
+### Searchs for POM files

```

Comment:
Should have better organization.
- Move this section to the end of "Basic View".   
- Change the title to "Support POM files with customized name"